test/functional/Java8andUp/src/org/openj9/test/attachAPI/AttachApiTest.java

```diff
@@ -200,5 +215,26 @@ protected boolean checkLoadAgentOutput(String errOutput,
 		}
 		return outputLines;
 	}
+
+	/**
+	 * Find an executable in the jdk directory and fail if it is missing.
+	 * @param commandRoot Base name of the command, without ".exe" 
+	 */
+	protected void getJdkUtilityPath(final String commandRoot) {
+		char fs = java.io.File.separatorChar;
+		final String commandWithSuffix = 
+				PlatformInfo.isWindows() ? commandRoot + ".exe": commandRoot; //$NON-NLS-1$
```

Comment:
`fs` appears to be unused: please remove it.
formatting: spaces around operators (`:` here).
Please be consistent about the use of `//$NON-NLS-n$`; the comment appears twice below but there are more string literals without the comment.

Reply:
Thanks. NLS warnings are suppressed in the file, so I will leave out the annotations for now.  I will fix them in a separate change set.